doc/faq.rst

```diff
@@ -181,86 +181,59 @@ Using Spyder
 .. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?
    :container: + dropdown-id-using-existing-environment
 
-   To work with an existing environment in Spyder, you need to change Spyder’s default Python interpreter.
-   To do so, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
+   To work with an existing environment in Spyder, change the default Python interpreter for new :doc:`/panes/ipythonconsole`\s to point to this environment.
 
-   .. image:: /images/faq/faq-change-environment.png
-      :alt: Change default environment in Preferences option in status bar
-
-   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section.
+   To do so, open the :guilabel:`Python interpreter` section of Spyder's preferences (:menuselection:`Tools --> Preferences`, or :menuselection:`Spyder --> Preferences` on macOS).
    Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment.
-   If its not listed, use the text box or the :guilabel:`Select file` button to enter the path to the Python interepreter you want to use.
-   See the :doc:`/panes/ipythonconsole` for more information.
+   If it's not listed, see :ref:`the note below <faq-env-not-listed>`.
 
    .. image:: /images/faq/faq-python-interpreter.png
       :alt: Preferences showing changing Python interpreter
 
-   Click :guilabel:`Restart kernel` in the :guilabel:`Consoles` menu for this change to take effect.
+   .. _faq-env-not-listed:
+
+   .. note::
+
+      If you installed Miniconda (or another Conda-based distribution) to a non-default path, or are using a virtual environment managed by a tool other than ``pyenv``, your environments likely won't be listed.
+
+      Instead, use the text box or the :guilabel:`Select file` button to enter the path to the Python interpreter you want to use.
+      You can find this path by activating the venv or Conda env you want to use in your terminal/Anaconda Prompt, and running the shell command:
```

Comment:
```suggestion
      You can find this path by activating the venv or Conda env you want to use in your terminal or the Anaconda Prompt, and running the command:
```

Small suggestion to make this sentence a bit easier to read.

Reply:
Sure; one further followup to clarify that the Anaconda Prompt should be used on Windows, and avoid a hard to read and potentially confusing double-or:

```suggestion
      You can find this path by activating the venv or Conda env you want to use in your terminal (Anaconda Prompt on Windows), and running the command:
```